systemtests/scripts/functions

```diff
@@ -473,6 +473,28 @@ stop_bareos()
    fi
 }
 
+
+run_python_unittests()
```

Comment:
```suggestion
run_python_unittests()
```
Perhaps Philipp should review this function because he is currently reviewing the unit-test code, especially the path arguments.

Reply:
If the sourcecode is checked out in a long subpath, the tests partly failed because of the limited length of some resources in bareos. To avoid these problems, absolute paths have been changed to relative paths where this problem showed up. 

As the pathes used here seem only to be stored in the environment I don't think that we will have these problems.